common/txmgr/confirmer.go

```diff
@@ -102,6 +103,10 @@ var (
 	}, []string{"chainID"})
 )
 
+type confirmerHeadTracker interface {
```

Comment:
Personally don't think this is necessary, so I will change to import `logpoller.HeadTracker` instead of creating a new local interface to reduce redundency, and keep consistency with ([chainReader](https://github.com/smartcontractkit/chainlink/blob/15dc74cabd3a83041ca97df54ea0fbb7e76e2a0a/core/services/relay/evm/chain_reader.go#L36), [methodBinding](https://github.com/smartcontractkit/chainlink/blob/15dc74cabd3a83041ca97df54ea0fbb7e76e2a0a/core/services/relay/evm/method_binding.go#L31)).